recipes/gnu-config/all/conanfile.py

```diff
@@ -15,7 +15,7 @@ class GnuConfigConan(ConanFile):
     topics = ("gnu", "config", "autotools", "canonical", "host", "build", "target", "triplet")
     license = "GPL-3.0-or-later", "autoconf-special-exception"
     package_type = "build-scripts"
-    os = "arch", "compiler", "build_type", "arch"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
Thanks @chausner  - looking at the recipe this line can be removed completely, as there doesnt seem to be anything in the build process that require any of the settings.